src/trackingModules/modeling/ModelingTracking.js

```diff
@@ -0,0 +1,68 @@
+export default class ModelingTracking {
+  constructor(eventBus, bpmnJSTracking, elementRegistry, selection) {
+    this._eventBus = eventBus;
+    this._bpmnJSTracking = bpmnJSTracking;
+    this._elementRegistry = elementRegistry;
+    this._selection = selection;
+
+    this._eventBus.on(
+      'commandStack.elements.create.postExecuted',
+      this.trackElementsCreate.bind(this)
+    );
+
+    this._eventBus.on(
+      'commandStack.shape.append.postExecuted',
+      this.trackElementAppend.bind(this)
+    );
+
+    this._eventBus.on(
+      'commandStack.shape.replace.postExecuted',
+      this.trackElementReplace.bind(this)
+    );
```

Comment:
[q] is it relevant for the tracking if we track the event before/after the `postExecuted` callback is called? If this is relevant, we should add a priority to the subscription. If we only care that it happened at one point, we can keep it as is. 

Assumption: no `postExecuted` callback modifies the relevant context entries.

Reply:
We care that it happened, not necessarily the exact moment. And so far I haven't encountered any issue with the context available